packages/cache/src/index.js

```diff
@@ -1,30 +1,35 @@
 // @flow
 import { StyleSheet } from '@emotion/sheet'
 import { type EmotionCache, type SerializedStyles } from '@emotion/utils'
-import Stylis from '@emotion/stylis'
+import {
+  serialize,
+  compile,
+  middleware,
+  rulesheet,
+  stringify,
+  compat,
+  prefixer
+} from '@emotion/stylis'
 import weakMemoize from '@emotion/weak-memoize'
-import { Sheet, removeLabel, ruleSheet } from './stylis-plugins'
+import { removeLabel, createUnsafeSelectorsAlarm } from './stylis-plugins'
 import type { StylisPlugin } from './types'
 
 let isBrowser = typeof document !== 'undefined'
 
 export type PrefixOption =
   | boolean
   | ((key: string, value: string, context: 1 | 2 | 3) => boolean)
-type StylisPlugins = StylisPlugin[] | StylisPlugin
 
 export type Options = {
   nonce?: string,
-  stylisPlugins?: StylisPlugins,
+  stylisPlugins?: StylisPlugin[],
   prefix?: PrefixOption,
```

Comment:
Am I correct in thinking that the prefix option is gone? And people will be able to pass in their own prefixer into plugins?

Reply:
> Am I correct in thinking that the prefix option is gone? And people will be able to pass in their own prefixer into plugins?

Yes, that's my current thinking and the way to customize prefixer - this includes removing it or changing what gets prefixed. You don't pass any special options, you just customize plugins however you want.

Reply:
This is also a good time to reevaluate what browsers should we support in the builtin prefixer. We could save some bytes both from the bundlesize (less prefixing logic) and from SSRed styles

Reply:
Happy to re-evaluate what prefixes we support by default. (and ideally we could offer a prefixer that's equivalent for anyone who cares about those browsers)

Reply:
> Happy to re-evaluate what prefixes we support by default.

Do you have any possible criteria in mind? I would be in favor of supporting flex prefixing for IE11 and some reasonable market share criteria (1%?) for everything else.

> (and ideally we could offer a prefixer that's equivalent for anyone who cares about those browsers)

This is basically what the currently included prefixer does - it's supporting quite a bit of properties. I think it actually supports a little bit more than what was included in Stylis v3.

Reply:
> Do you have any possible criteria in mind? I would be in favor of supporting flex prefixing for IE11 and some reasonable market share criteria (1%?) for everything else.

That sounds good to me, though maybe create an issue and tweet about it to get some more perspectives?